rust-src/concordium_base/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased changes
 
+- Add extern function `free_array_len_cap` such vector pointers can be released and deprecated `free_array_len`
+  since it didn't account for capacity size.
```

Comment:
`ffi` stuff is internal and we make no guarantees about it. It should also not go into the public changelog.

Reply:
removed